django/bosscore/views/views_resource.py

```diff
@@ -656,7 +656,8 @@ def get(self, request, collection, experiment, channel):
                                   962, 963, 964, 965, 966, 967, 975, 976, 977, 978, 979, 980, 981, 982, 983, 984, 985,
                                   986, 987, 988, 989, 993, 995, 997, 1004, 1033, 1034, 1035, 1037, 1039, 1040, 1041,
                                   1042, 1043, 1044, 1045, 1046, 1047, 1062, 1066, 1067, 1068, 1069, 1070, 1071, 1072,
-                                  1073, 1074, 1080, 1083] or request.user.has_perm("read", channel_obj):
+                                  1073, 1074, 1080, 1083, 1090, 1091, 1092, 1093, 1094, 1095, 1096, 1097, 1098, 1099,
```

Comment:
Now that this is _this_ huge, we should put this in a `frozenset` for faster lookup and define the `frozenset` in a single file so we don't have to worry about mismatches between these two files.

https://docs.python.org/3.8/library/stdtypes.html#set-types-set-frozenset

Reply:
Updated to use a frozenset.